fuzz_plugin.py

```diff
@@ -70,16 +71,22 @@ def fuzz_plugin(
     else:
         enabled_features = enabled_features.split(",")
 
-    assert all(
-        [letter in string.ascii_letters + string.digits + "-_" for letter in slug]
-    )
-
-    plugin_info_dict = None
-    print("Looking for", slug)
-    plugin_info_dict = requests.get(
-        f"https://api.wordpress.org/plugins/info/1.2/?action=plugin_information"
-        f"&request[slug]={slug}"
-    ).json()
+    if slug.endswith('.zip') and os.path.exists(slug):
```

Comment:
I think if this variable could contain both a slug and a path, it should be named differently - maybe even "slug_or_path"